### DISCUSSION TITLE
Obscure "Type error" occurring on props of a React component

### DISCUSSION THREAD

**Discussion by hayden2114** [pmndrs/jotai#1292] [Q&A]
I've encountered a very obscure error in my react project and I'm not sure what it could be since the error is just `TypeError: Type error`. I've posted more details on stackoverflow: https://stackoverflow.com/questions/73052846/obscure-type-error-occurring-on-props-of-a-react-component

This could have nothing to do with jotai but the prop being passed in is a jotai atom so I thought it could be related. Has anyone encountered an error like this before using jotai atoms?

Using react v17 and jotai v1.4.8

**Comment by dai-shi** [comment]
Whether it's related with jotai or not, if you try to reproduce your issue with minimal code, you will find something.
Even if you don't get it, with such minimal reproduction, someone would be easier to help.
CodeSandbox is preferred if possible.

**Reply by hayden2114** [reply]
Thanks for the quick response. 

I have tried to reproduce it to no luck. Even on my live site, with thousands of visitors per day, it only happens every few days.
